src/newmagic.cpp

```diff
@@ -1788,6 +1995,18 @@ void mob_magic(struct char_data *ch)
         if (!affected_by_spell(FIGHTING(ch), SPELL_IGNITE) && !ch->points.fire[0])
           spell = SPELL_IGNITE;
         break;
+      case 13:
+        spell = SPELL_LASER;
+        break;
+      case 14:
+        spell = SPELL_STEAM;
+        break;
+      case 15:
+        spell = SPELL_THUNDERBOLT;
+        break;
+	  case 16:
```

Comment:
Nit: Indentation- this should be in line with the other case statements.

Reply:
Realigned, I hope.